stable/keel/templates/deployment.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
   annotations:
-    kubernetes.io/service-account.name: keel
+    kubernetes.io/service-account.name: {{ template "keel.serviceAccountName" . }}
```

Comment:
Why would this annotation be necessary?

Reply:
For older compatibility but I guess it can be removed.